src/Narochno.Slack/SlackClient.cs

```diff
@@ -43,13 +46,68 @@ public SlackClient(HttpClient httpClient, SlackConfig slackConfig)
             message.Channel = message.Channel.Fallback(slackConfig.Channel);
             message.Emoji = message.Emoji.Fallback(slackConfig.Emoji);
 
-            var json = JsonConvert.SerializeObject(message, serializerSettings);
+            string json = JsonConvert.SerializeObject(message, serializerSettings);
 
-            var response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));
+            HttpResponseMessage response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));

```

Comment:
I disagree with the reasoning as, to me, var makes things less verbose and more readable.  Basically the arguments for removing parentheses and crap that F#/Scala/whatever make.  Your contexts are small enough to understand each var easily enough.  I’ve changed my style to use underscores for fields because most people do that.

That said, I’m not that bothered if you want to do that.  We should have a resharper settings file for this stuff.